composer.json

```diff
@@ -16,14 +16,22 @@
     ],
     "require": {
         "php": "^7.4|^8.0",
+        "illuminate/contracts": "^8.0",
         "spatie/laravel-package-tools": "^1.1",
-        "illuminate/contracts": "^8.0"
+        "stripe/stripe-php": "^7.69",
+        "tipoff/support": "^1.0"
     },
     "require-dev": {
         "orchestra/testbench": "^6.0",
         "phpunit/phpunit": "^9.3",
         "spatie/laravel-ray": "^1.9"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/tipoff/support"
+        }
+    ],
```

Comment:
I would prefer using the main composer version on pakagist.org. I'll do a commit to remove this since I already started issuing Releases on the support package:

- https://packagist.org/packages/tipoff/support